website/docs/docs/dbt-versions/release-notes/08-May-2023/run-details-and-logs-improvements.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Run details and logs usability and design improvements"
+sidebar_label: "Update: Improvements to run details and logs"
+sidebar_position: 3
+tags: [May-26-2023, Scheduler]
+---
+
+New usability and design improvements to the run details and logs in dbt Cloud are now available. The ability to triage errors in logs is a big benefit of using dbt Cloud's job and scheduler functionality. These updates help make the process of finding root causes much easier.
```

Comment:
```suggestion
New usability and design improvements to the run details and logs in dbt Cloud are now available. The ability to triage errors in logs is a big benefit of using dbt Cloud's job scheduler functionality. These updates help make the process of finding root cause much easier.
```

Reply:
got it! suggest adding the definite article:

```suggestion
New usability and design improvements to the run details and logs in dbt Cloud are now available. The ability to triage errors in logs is a big benefit of using dbt Cloud's job and scheduler functionality. These updates help make the process of finding the root cause much easier.
```